actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -709,6 +717,14 @@ def convert_options_to_data_attributes(options, html_options)
           end
         end
 
+        def url_target(name, options)
```

Comment:
Looks like `url_for` gets called twice now. Once in this method, then once on its return value on line 212. Is that necessary?

Reply:
I'll check.

Reply:
Yeah not sure why I was calling `url_for` twice there. But it's fixed now.